motioneye/handlers/action.py

```diff
@@ -36,6 +36,13 @@ async def post(self, camera_id, action):
             raise HTTPError(404, 'no such camera')
 
         local_config = config.get_camera(camera_id)
+        # block access to admin-only cameras for non-admin users
+        if (
+            local_config
+            and local_config.get('@admin_only')
+            and self.current_user != 'admin'
+        ):
+            raise HTTPError(403, 'access denied')
```

Comment:
The second argument is a server-side log message, not the one returned to the client. And it is printed only as debug log: https://www.tornadoweb.org/en/stable/_modules/tornado/web.html#HTTPError
For debugging, it makes sense to add more more details with this message:
```suggestion
            raise HTTPError(403, f'access denied to admin-only camera "{camera_id}" for action "{action}"')
```